Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -341,7 +347,7 @@ CEXIETHERNET::BuiltInBBAInterface::TryGetDataFromSocket(StackRef* ref)
 void CEXIETHERNET::BuiltInBBAInterface::HandleTCPFrame(const Common::TCPPacket& packet)
 {
   const auto& [hwdata, ip_header, tcp_header, ip_options, tcp_options, data] = packet;
-  sf::IpAddress target;
+  std::optional<sf::IpAddress> target;
```

Comment:
I'm guessing `sf::IpAddress` no longer has a default ctor?

It looks like `target` can just be defined down where it's actually assigned and used instead of at the top of the function here to avoid `std::optional`.

Similar situation for other `sf::IpAddress` changes.